modules/git/blame_test.go

```diff
@@ -112,9 +112,7 @@ func TestReadingBlameOutput(t *testing.T) {
 		},
 		{
 			"ce21ed6c3490cdfad797319cbb1145e2330a8fef",
-			[]string{
-				"// Copyright 2016 The Gitea Authors. All rights reserved.",
-			},
+			[]string{"// Copyright 2016 The Gitea Authors. All rights reserved."},
 		},
 		{
```

Comment:
The CI failure appears to be in `#124`, which is interesting, as it suggests another bug with the backport.

Reply:
What's interesting is that the test blame file is not actually a valid porcelain output from blame...

Reply:
Aha! the there's a whitespace change in the commit - I've reverted that.